docs/BuildOnLinuxOSX.md

```diff
@@ -75,6 +75,10 @@ The environment variable `$pythonLocation` may be used to specify the base direc
 
 After the above commands succeed, an `onnx-mlir` executable should appear in the `Debug/bin` or `Release/bin` directory.
 
+### Enable CPU Optimizations
```

Comment:
maybe just clarify that CPU optimization are here to make the compiler run faster. That may clarify some confusion about what is accelerated

Reply:
good point, I added "To make the compiler run faster (without any affect on the generated code)"